From 01922a980ff4f055fad3a3570d971f411082bb0c Mon Sep 17 00:00:00 2001
From: Xiang Wang <xiang.a.wang@intel.com>
Date: Thu, 4 Jun 2015 14:46:03 +0800
Subject: [PATCH] i2c: designware: reduce log level on illegal access

dev_WARN was used to track who was accessing i2c
at wrong time. But printing these logs to console
wastes time. After we fixed wrong accesses in other
device drivers, the traces are not necessary.

Change-Id: Ic6e7d981fa817ae59553e708415e76f8d7238b12
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-26433
Signed-off-by: Xiang Wang <xiang.a.wang@intel.com>
---
 drivers/i2c/busses/i2c-designware-core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/i2c/busses/i2c-designware-core.c b/drivers/i2c/busses/i2c-designware-core.c
index 49fdf86cb21e..5c07e7f0b270 100644
--- a/drivers/i2c/busses/i2c-designware-core.c
+++ b/drivers/i2c/busses/i2c-designware-core.c
@@ -706,7 +706,7 @@ i2c_dw_xfer(struct i2c_adapter *adap, struct i2c_msg msgs[], int num)
 
 	if (dev->status & STATUS_SUSPENDED) {
 		mutex_unlock(&dev->lock);
-		dev_WARN(dev->dev, "i2c xfer after suspend!\n");
+		dev_dbg(dev->dev, "i2c xfer after suspend!\n");
 		return -EIO;
 	}
 
-- 
1.9.1

